Gruntfile.js

```diff
@@ -79,19 +79,49 @@ module.exports = function(grunt) {
     }
   }
 
+  var combineCustomJS = {
```

Comment:
glad you're adding this!

Reply:
haha. sorry for the inconveniences. I also had enough of it.